certif/app/routes/authenticated/sessions/list.js

```diff
@@ -1,10 +1,15 @@
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
 export default Route.extend({
+  currentCertificationCenter: service(),
 
   model() {
-    const store = this.get('store');
-    return store.findAll('session');
+    return this.get('currentCertificationCenter').certificationCenter
```

Comment:
Une approche this.store.query('sessions', { params }) me parait plus résiliente pour la suite. (on a eu ce problème dans orga et admin) il faut en discuter.

Reply:
Discussion effectuer, on le fait en mob :)